src/ConcatenateMsg.chpl

```diff
@@ -140,7 +140,7 @@ module ConcatenateMsg
                             start += o.size;
                         }
                         cmLogger.debug(getModuleName(),getRoutineName(),getLineNumber(),
-                                         "created concatenated pdarray: %t".format(st.lookup(rname)));
+                                         "created concatenated pdarray: %s".format(rname));
```

Comment:
could we change this to st.attrib(valName)? That will provide the info I think would be good for debugging purposes.